### ISSUE TITLE
Ability to use it in android

### ISSUE THREAD

**Issue by TheBricktop** [gespona/depthai-unity-plugin#1]
I've seen on the depth ai discord some movement towards building support for android platform, do you see it possible to add android support to Your awesome plugin in a forseeable future?

**Comment by gespona** [comment]
Hi ! Yes I'm following too and it's really exciting. Definitely, as soon as it's bit more stable and clear I will add support for the plugin ! Btw, I'm close to do big release on AssetStore with many other predefined models ready-to-use.

And you can find me on discord too (gespona#6853)

**Comment by TheBricktop** [comment]
Thats great to hear, definitely im following the development of anything oak-d centered especially when intel is bagging almost entire realsense branch

**Comment by gespona** [comment]
Yes bit disappointed .. just one year ago I bought my D-435 🤷‍♂️  .... along OAK-D Wifi ... one door closing .. another opening :)

**Comment by TheBricktop** [comment]
We bought about 50 d450s along with 50 correspondend d4 VPUs for custom cases and that hurts.

**Comment by gespona** [comment]
Yes .. really think Intel would manage better the EOL taking into account the amount of people working with realsense product line .....

**Comment by TheBricktop** [comment]
Seems like IbaiGordo has made significant improvements in that matter
https://github.com/ibaiGorordo/depthai-android-unity-example

**Comment by TheBricktop** [comment]
No rush,  im just very pleased at the speed of the development of new features.
